pages/company/visualizations/commit_domains.py

```diff
@@ -0,0 +1,212 @@
+from pages.utils.job_utils import nodata_graph
+import time
+import datetime as dt
+
+PAGE = "company"
+VIZ_ID = "commit-domains"
+
+paramter_1 = "company-contributions-required"
+
+
+gc_commit_domains = dbc.Card(
+    [
+        dbc.CardBody(
+            [
+                html.H3(
+                    "Commit Activity by Domain",
+                    className="card-title",
+                    style={"textAlign": "center"},
+                ),
+                dbc.Popover(
+                    [
+                        dbc.PopoverHeader("Graph Info:"),
+                        dbc.PopoverBody(
+                            "This graph shows the proportions of email domains that contributors used to make commits.\n\
+                            e.g. johndoe@gmail.com as a commit author would contribute to the count of gmail.com once.\n\
+                            NOTE: the domain might not respresent if it is done as an individual or company."
+                        ),
+                    ],
+                    id=f"{PAGE}-popover-{VIZ_ID}",
+                    target=f"{PAGE}-popover-target-{VIZ_ID}",  # needs to be the same as dbc.Button id
+                    placement="top",
+                    is_open=False,
+                ),
+                dcc.Loading(
+                    dcc.Graph(id=VIZ_ID),
+                ),
+                dbc.Form(
+                    [
+                        dbc.Row(
+                            [
+                                dbc.Label(
+                                    "Contributions Required:",
+                                    html_for=f"{PAGE}-{paramter_1}-{VIZ_ID}",
+                                    width={"size": "auto"},
+                                ),
+                                dbc.Col(
+                                    [
+                                        dbc.Input(
+                                            id=f"{PAGE}-{paramter_1}-{VIZ_ID}",
+                                            type="number",
+                                            min=1,
+                                            max=100,
+                                            step=1,
+                                            value=10,
+                                            size="sm",
+                                        ),
+                                    ],
+                                    className="me-2",
+                                    width=2,
+                                ),
+                            ],
+                            align="center",
+                        ),
+                        dbc.Row(
+                            [
+                                dbc.Col(
+                                    dcc.DatePickerRange(
+                                        id=f"{PAGE}-date-picker-range-{VIZ_ID}",
+                                        min_date_allowed=dt.date(2005, 1, 1),
```

Comment:
Is this the earliest commit in Augur? Are there any commits older than this? e.g. should we hard-code to an earlier date if we could possible be screening out earlier stuff.

Reply:
This is an arbitrary date so open to changing it. Git was released in 2005 so this should cover it